xapian-letor/ranker/ranker.cc

```diff
@@ -164,7 +164,12 @@ load_relevance(const std::string & qrel_file)
 	    if (*end) {
 		throw LetorParseError("Could not parse relevance label in Qrel file at line:" + str(qrel_count));
 	    }
-	    qrel1[token[0]].insert(make_pair(token[2], label));
+	    // Map the file name in the qrel file, to the doc-id
+	    Xapian::PostingIterator db_iter = (db.postlist_begin("U/" + token[2]));
```

Comment:
There's no need to put parentheses around the value here, and it makes it less readable if anything (because you end up with stacked `))` at the end which means the reader has to pause to match them.